src/Animals/Animals.js

```diff
@@ -4,12 +4,12 @@ import DogCard from "../DogCard/DogCard"
 
 const Animals = ({dogs}) => {
   const dogCards = dogs.map((dog, index) => {
-    const uniqueValue = Date.now()
+    const value = Date.now()
     return (
       <DogCard
-      id={uniqueValue} 
-      dogPic={dog[index]}
-      key={uniqueValue}
+      id={value + index}
+      dogPic={dog}
+      key={value + index}
```

Comment:
Where you getting duplicate `Date.now()?` from the cards all being made at the same time? This seems like a fine solution. You could also look into using a UUID. I think installing a UUID npm package and using it doesn't require a steep learning curve.

Reply:
I think it was because I was making uniqueValue a const variable outside the map function at one point, which no longer made it unique, haha. I'll look into the UUID npm package and implement it for fun, if I have the time. Thank you so much, Anna!